From 5521004cbe69a6f7f991462eb623d1747d2de14b Mon Sep 17 00:00:00 2001
From: Adnan Begovic <adnan@cyngn.com>
Date: Fri, 8 May 2015 14:42:25 -0700
Subject: [PATCH 04/10] build/core: Create means of ignoring subdir layer for
 packages.

  An external resource package that acts as a secondary framework
  resource should be presented in system/framework similarily to
  the framework-res module.

Change-Id: Ie4110a184cd7262035110a6a04bb7ea91e7a42b0
---
 core/base_rules.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/base_rules.mk b/core/base_rules.mk
index 046d911b7..ed08d05ec 100644
--- a/core/base_rules.mk
+++ b/core/base_rules.mk
@@ -305,6 +305,7 @@ ifneq (true,$(LOCAL_UNINSTALLABLE_MODULE))
   ifeq ($(LOCAL_NO_STANDARD_LIBRARIES),true)
   # Neither do Runtime Resource Overlay apks, which contain just the overlaid resources.
   else ifeq ($(LOCAL_IS_RUNTIME_RESOURCE_OVERLAY),true)
+  else ifeq ($(LOCAL_IGNORE_SUBDIR),true)
   else
     my_module_path := $(my_module_path)/$(LOCAL_MODULE)
   endif
-- 
2.17.1

